XamlAnimatedGif.Shared/Interfaces/ICacheHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿#region
+
+using System.IO;
+using System.Threading.Tasks;
+
+#endregion
+
+namespace XamlAnimatedGif.Interfaces
+{
+    internal interface ICacheHelper
```

Comment:
Is it really necessary to split the functionality between 3 interfaces? Seems a bit overkill to me...